### DISCUSSION TITLE
Thank you for making this extension

### DISCUSSION THREAD

**Discussion by ziontee113** [mcmonkeyprojects/sd-infinity-grid-generator-script#82] [General]
Hi. I stumbled across this extension while searching for "infinity" in the extensions tab. This extension feels very promising to me since I use SD like a gambling machine :sweat_smile:.
After a few days of procrastinating, I finally decided to try this myself.

![Screenshot from 2023-08-04 18-49-53](https://github.com/mcmonkeyprojects/sd-infinity-grid-generator-script/assets/102876811/862aa7ea-cf64-4311-8900-3e5c292e1ab0)

This is the first grid I'm about to generate, don't know how long that's gonna take :sweat_smile: 
Can't wait to see how it goes.

**Comment by ziontee113** [comment]
Well, it took 4+ hours. And now I want to test out more variations at higher steps.
Luckily for me, it seems like I can just add more values to the steps, and the extension will only generate new ones?
I'm so very glad.

**Reply by mcmonkey4eva** [reply]
Yep, you can for the most part just toss new ones in
